lib/base/utility.cpp

```diff
@@ -1946,3 +1938,20 @@ String Utility::GetIcingaDataPath(void)
 }
 
 #endif /* _WIN32 */
+
+void Utility::RenameFile(const String& source, const String& destination)
+{
+#ifdef _WIN32
+	if (remove(destination.CStr()) && errno != ENOENT)
```

Comment:
1. Keep _unlink as is. This commit is about code cleanup, not changing things.
2. Tests are required, since you're changing the behaviour with _unlink/rename here in basically any feature.